dxr/plugins/xpidl/tests/test_menus_and_filters/code/nsIDownloadHistory.idl

```diff
@@ -0,0 +1,65 @@
+/* -*- Mode: C++; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*-
```

Comment:
I think the file looks bigger than it actually is because of the doc comments; it has 2 methods, 1 attribute, and 1 const member. I'll go ahead and shorten the file while keeping the same interesting points.